.ci/Jenkinsfile-tests

```diff
@@ -55,6 +56,20 @@ pipeline {
                 '''
             }
         }
+        stage('Style checking') {
+             steps {
+                dir("server"){
+                    sh '''
+                        rm -rf $INMANTA_STYLE_CHECK_ENV
+                        python3 -m venv $INMANTA_STYLE_CHECK_ENV
+                        source $INMANTA_STYLE_CHECK_ENV/bin/activate
+                        pip install --upgrade pip
+                        make pep8
+                        make format
```

Comment:
Executing `make format` is not useful here. It's a command used to fix the formatting, but if the formatting wasn't right, the previous command would have failed.